app/component/itinerary/NaviCardExtension.js

```diff
@@ -21,10 +25,51 @@ const NaviCardExtension = ({ leg }, { config }) => {
     destination.className = 'place';
     destination.name = place;
   }
+
+  if (legType === 'in-transit') {
+    const { intermediatePlaces } = leg;
+    const now = new Date();
+    const idx =
+      intermediatePlaces.findIndex(
+        p =>
+          new Date(
+            p.arrival.estimated?.time || p.arrival.scheduledTime,
+          ).getTime() > now.getTime(),
+      ) ?? -1;
+
+    const count = idx > -1 ? intermediatePlaces.length - idx : 0;
+    const stopCountElement = <span className="realtime"> {count}</span>;
```

Comment:
I guess plain `stopCount` isa bit  simpler. We don't have a convention to name jsx snippets as elements.